src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/TrackerObjectManager.NativeAot.cs

```diff
@@ -137,9 +137,9 @@ public static void EndReferenceTracking()
 
         public static unsafe void RegisterGCCallbacks()
         {
-            delegate* unmanaged<int, void> gcStartCallback = &GCStartCollection;
-            delegate* unmanaged<int, void> gcStopCallback = &GCStopCollection;
-            delegate* unmanaged<int, void> gcAfterMarkCallback = &GCAfterMarkPhase;
+            delegate* unmanaged[Fastcall]<int, void> gcStartCallback = &GCStartCollection;
```

Comment:
Can we change the unmanaged side to use the default calling convention instead?

x86 custom calling conventions are pain. It is best to stick to the default calling convention where possible.

Reply:
We have two different callouts https://`github.com/dotnet/runtime/blob/90dfc04d8f863e053e42edf0992c1b63398b12fb/src/coreclr/nativeaot/Runtime/RestrictedCallouts.h#L100-L101 . One of them is FastCall and the other one is default. I would make both of them default.